gems/smithy/lib/smithy/anvil/views/client/endpoint_parameters.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module Smithy
+  module Anvil
+    module Views
+      module Client
+        # @api private
+        class EndpointParameters < View
+          def initialize(plan)
+            @plan = plan
+            @model = plan.model
+            # TODO: I reference the endpoint trait ids in many places.
+            # Should we have a constant somewhere for all of them or is it fine to just use the ids as is?
```

Comment:
I think just the string is fine. Constant doesn't give us anything useful and it's private API.

Reply:
If there is a file that contains shared logic between endpoints views, maybe constants there is preferred to keep all references in a single place.